.gitignore

```diff
@@ -8,3 +8,4 @@ oc
 yq
 openshift-clients/
 podman-remote/
+.swp
```

Comment:
Might be worth matching `.sw?` https://unix.stackexchange.com/questions/326707/vim-what-are-all-the-possible-swapfile-extensions